README.md

```diff
@@ -0,0 +1,7 @@
+# browse-activity
+Browser is a computer program with a graphical user interface for displaying HTML files, used to navigate the World Wide Web.
+A browse-activity is to browse the internet WebKit2 on GTK+ on Sugar Toolkit.
+This is a python generated activity to browse the internet using webkit on gtk+ on sugar toolkit
+Sugar includes a Mozilla-Firefox-derived (Gecko) Web Browser Activity. 
+<p>This browser includes HTML, XHTML, Javascript and SVG support. 
+we can set up simple web-servers on a laptop using Python, or children may access an application hosted on a public web server </p>
```

Comment:
Perhaps:

Browser can be used with a simple, local web server can be set up using Python or it may be used to access pages hosted on a public web server when available.